js/test/components/operatorSignIn/operatorSignInModal.test.tsx

```diff
@@ -1,14 +1,41 @@
 import { OperatorSignInModal } from "../../../components/operatorSignIn/operatorSignInModal";
+import { nfcSupported } from "../../../util/nfc";
 import { render } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 
+jest.mock("../../../util/nfc", () => ({
+  __esModule: true,
```

Comment:
Same as above with `__esModule` here (and the rest of the usages in this file)

Reply:
0118b17

Reply:
And also 57d8ca2.